ui/app/AppMain.qml

```diff
@@ -47,7 +47,20 @@ Item {
             browserLayoutContainer.item.openUrlInNewTab(link);
         }
         onOpenChooseBrowserPopup: {
-            appMain.openPopup(chooseBrowserPopupComponent, {link: link})
+            Global.openPopup(chooseBrowserPopupComponent, {link: link});
+        }
+        onOpenPopupRequested: {
+            const popup = popupComponent.createObject(appMain, params);
+            popup.open();
+            return popup;
+        }
+    }
+
+    Loader {
+        id: popupLoader
+        active: !!item
+        Component.onCompleted: {
+            Global.popupLoader = this;
```

Comment:
dont see a property with this name in Global.qml?

Reply:
right, it was a leftover, removed!